### PR TITLE
GDS 179 | make rule for inserting reports to DSDocs

### PR DESCRIPTION
**Pull request by recursive-automata**
## Change Notes
* Added a configuration point on experiment initialization for location of DSDocs site.
* Added a Make rule to render and insert the contents of `notebooks/report/` into DSDocs.

## Jira Issues
- [GDS-179](https://55places.atlassian.net/browse/GDS-179)
- [GDS-160](https://55places.atlassian.net/browse/GDS-160)

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/Makefile [hunk 1]
```diff
@@ -143,6 +144,13 @@ enter_docker:
 test_environment:
 	$(PYTHON_INTERPRETER) test_environment.py
 
+## Render and insert report notebooks into DSDocs Jekyll site.
+insert_into_dsdocs:
+	$(PYTHON_INTERPRETER) "${DSDOCS_DIR}/scripts/render_and_insert_all.py" \
+	    ./notebooks/report/ \
```
Comment (jillrobinson) [inline {{ cookiecutter.repo_name }}/Makefile]: I believe one of our old JIRA tickets (or at least, @davidzornek has mentioned it before) includes removing the notebooks/report and notebooks/exploratory subdirectories and just having notebooks/. If that is still what we want to do, should we simply remove those subdirectories as part of this PR and refer to notebooks/ only?

Comment (davidzornek) [reply {{ cookiecutter.repo_name }}/Makefile]: I think so. Technically out of scope, but perhaps tiny enough that @recursive-automata will let it slide. :P

**Review by davidzornek — CHANGES_REQUESTED** [review]
Just the thing about removing subdirectories to the notebooks directory.

File: {{ cookiecutter.repo_name }}/Makefile (hunk 1 above)
Comment (recursive-automata) [reply {{ cookiecutter.repo_name }}/Makefile]: Perhaps so. :)

**Review by jillrobinson — APPROVED** [review]

**Review by davidzornek — APPROVED** [review]

**Pull request merged**